CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+- Support provider-side previews. These changes will admit more known property values during
```

Comment:
I would perhaps change this whole thing to something more like:

> Improve the accuracy of previews by allowing providers to participate in determining what the impact of a change will be on output properties.  Previously, Pulumi previews conservatively assumed that any output-only properties changed their values when an update occurred.  For many properties, this was guaranteed to not be the case (because those properties are immutable for example), and by suggesting the value might change, this could lead to the preview suggesting additional transitive updates of even replaces that would not actually happen during an update.  Pulumi now allows the provider to specify the details of what properties will change during a preview, allowing them to expose more accurate provider-specific knowledge.  This change is less conservative than the previous behavior, and so in case it causes preview results which are not deemed correct in some case - the `PULUMI_DISABLE_PROVIDER_PREVIEW` flag can be set to a truthy value to go back to the previous and more conservative Pulumi preview behavior.